django_rest_resetpassword/serializers.py

```diff
@@ -10,16 +12,17 @@
 
 
 class EmailSerializer(serializers.Serializer):
-    email = serializers.EmailField(required=False)
-    username = serializers.CharField(required=False)
-
-    def validate(self, data):
-        """
-        Check email and username can't be provided together.
-        """
-        if not (bool(data.get("username")) ^ bool(data.get("email"))):
-            raise serializers.ValidationError("Invalid Inputs, only one field required.")
-        return data
+    email_or_username = serializers.CharField()
+
+    def validate_email_or_username(self, value):
+        print(get_password_reset_lookup_fields)
+        if get_password_reset_lookup_fields() == ["email"]:
```

Comment:
you are comparing lists, that won't work. don't you want to test membership instead?